### DISCUSSION TITLE
Replace Jetty with Armeria

### DISCUSSION THREAD

**Discussion by kezhenxu94** [apache/skywalking#8666] [Ideas]
Since we've been using Armeria as our ElasticSearch HTTP client, and Armeria can serve as an HTTP server, I'm considering whether we could remove Jetty dependencies by using Armeria as our HTTP endpoint server, this includes the agent report HTTP protocol and the GraphQL query protocol.
We can reduce more than 7 dependencies from this.

**Comment by wu-sheng** [comment]
I am definitely supporting this.

**Comment by wu-sheng** [comment]
@wankai123 I think you asked this, right?

**Reply by wankai123** [reply]
yes, I support this. But should we do some evaluation work before? Whether there are other benefits except to reduce the dependencies. At least not has negative impacts.
